css/properties/background-attachment.json

```diff
@@ -77,11 +77,21 @@
               "opera_android": {
                 "version_added": "14"
               },
-              "safari": {
-                "version_added": false
-              },
+              "safari": [
+                {
+                  "version_added": "14",
+                  "partial_implementation": true,
+                  "notes": "<code>local</code> is recognized but has no effect due to [a bug](https://bugs.webkit.org/show_bug.cgi?id=219324)."
+                },
+                {
+                  "version_added": "3.1",
+                  "version_removed": "14"
+                }
+              ],
               "safari_ios": {
-                "version_added": "4.2"
+                "version_added": "5",
+                "partial_implementation": true,
+                "notes": "<code>local</code> is recognized but has no effect."
```

Comment:
I'd suggest using the same note text for both Safari and Safari for iOS (since it's possible that BCD consumers may present them separately). Ideally, notes should stand alone.

Reply:
afaik `local` never worked in iOS safari unlike macOS safari, and the webkit bug ticket only mentions macOS, so i'm not sure if i really should link the bug ticket here for iOS.

Reply:
Ah, OK. This is fine then. Thank you!